src/app/settings/page.tsx

```diff
@@ -1,29 +1,110 @@
 'use client';
 
+import React from 'react';
 import { useRouter } from 'next/navigation';
-import supabase from '../../api/supabase/createClient';
+import supabase from '@/api/supabase/createClient';
+import { BackLink, H1, H4 } from '@/styles/text';
+import Button from '@/components/Button';
+import COLORS from '@/styles/colors';
+import SettingsSection from '@/components/SettingsSection';
+import { ContentContainer, PageContainer } from './styles';
 
 export default function Settings() {
   const { push } = useRouter();
+
   const handleSignOut = async () => {
     const { error } = await supabase.auth.signOut();
     if (error) {
       throw new Error(`An error occurred trying to sign out: ${error.message}`);
     }
     push('/login');
   };
-  const resetPassword = () => {
-    push('/reset-password');
-  };
+
+  // const resetPassword = () => {
+  //   push('/reset-password');
+  // };
 
   return (
-    <>
-      <button type="button" onClick={handleSignOut}>
-        Sign out
-      </button>
-      <button type="button" onClick={resetPassword}>
-        Reset Password
-      </button>
-    </>
+    <PageContainer>
+      <BackLink href="/cases">Back</BackLink>
+      <ContentContainer>
+        <H1>Your Profile</H1>
+
+        <SettingsSection
```

Comment:
let's cut the edit icon from here as per the figma -- reset email/password from settings is out of scope / potential stretch for rn